### ISSUE TITLE
Training parameters

### ISSUE THREAD

**Issue by ragondyn** [yujheli/ARN#3]
Hi, 

I would like to reproduce the results of your paper, also I've got some questions about the training process : 

- the images are said to be resized to 224x224x3, when your code resize images to 256x128x3. What is the good solution? Also, should we use Imagenet pretrained mean and std, or the solution described in the config file?

-You do not specify the number of epochs, neither the batch size used in the paper. Which values should we use?

-You also explain that EI is tuned only for the first few epochs, but I didn't see any mention of that in the code. Do you use some specific pre-training, different from imagenet pretraining?

-The learning rate is individually set for the different models : ET,EC,ES,DC is 1e-3, CS 2xe-3, where do we find this in the code?

-Would that be possible to provide the weights obtained at the end of the training process?

-Finally, which weighting values should we use for the different losses? The default values (written in the code) are different from the one given in the paper.

I'm fairly new to pytorch though, also I might have missed some points in the code, which would have answered some of my questions.

I thank you in advance for answering my questions.

**Comment by ThinkerPcw** [comment]
I think the results of this paper are fabricated. This code does not achieve what the authors say

**Comment by zaidbhat1234** [comment]
> Hi,
> 
> I would like to reproduce the results of your paper, also I've got some questions about the training process :
> 
> * the images are said to be resized to 224x224x3, when your code resize images to 256x128x3. What is the good solution? Also, should we use Imagenet pretrained mean and std, or the solution described in the config file?
> * You do not specify the number of epochs, neither the batch size used in the paper. Which values should we use?
> * You also explain that EI is tuned only for the first few epochs, but I didn't see any mention of that in the code. Do you use some specific pre-training, different from imagenet pretraining?
> * The learning rate is individually set for the different models : ET,EC,ES,DC is 1e-3, CS 2xe-3, where do we find this in the code?
> * Would that be possible to provide the weights obtained at the end of the training process?
> * Finally, which weighting values should we use for the different losses? The default values (written in the code) are different from the one given in the paper.
> 
> I'm fairly new to pytorch though, also I might have missed some points in the code, which would have answered some of my questions.
> 
> I thank you in advance for answering my questions.

Did you figure out the answers to the above questions? If Yes, Please share them with me.
I additionally faced an issue that though the output of decoder (Reconstructed image) is 3 channel, yet the reconstructed images are Black and White. Coloured images are ideally desired. 
Thank you in anticipation.

**Comment by muggledy** [comment]
> I think the results of this paper are fabricated. This code does not achieve what the authors say

Hello, how do you run his code? I rewrited the DataLoader due to the missing csv file, and got a very very low rank, can you give me some guidance, thank you very much.
